src/applications/hca/reducer.js

```diff
@@ -24,26 +31,29 @@ const initialState = {
 export function hcaEnrollmentStatus(state = initialState, action) {
   switch (action.type) {
     case FETCH_ENROLLMENT_STATUS_STARTED:
-      return { ...initialState, isLoading: true };
+      return { ...state, isLoadingApplicationStatus: true };
 
     case FETCH_ENROLLMENT_STATUS_SUCCEEDED: {
       const {
         parsedStatus: enrollmentStatus,
         applicationDate,
+        effectiveDate: enrollmentStatusEffectiveDate,
         enrollmentDate,
         preferredFacility,
       } = action.data;
       const isInESR =
         enrollmentStatus !== HCA_ENROLLMENT_STATUSES.noneOfTheAbove;
       return {
         ...state,
+        hasServerError: false,
```

Comment:
because the entire state isn't reset when the fetch begins, this success handler now needs to manually set a couple more values on the redux state